castai/resource_autoscaler.go

```diff
@@ -49,12 +89,381 @@ func resourceAutoscaler() *schema.Resource {
 				Description:      "autoscaler policies JSON string to override current autoscaler settings",
 				Optional:         true,
 				ValidateDiagFunc: validateAutoscalerPolicyJSON(),
+				Deprecated:       "use autoscaler_policy_overrides instead",
```

Comment:
can you update this message to include link to readme for examples?

Reply:
is it necessary, long messages will be reduce readability of this since it's part of the terraform plan

Reply:
I think it's a good practise to inform customer regarding next steps. I think we usually put these links we such changes.

Reply:
Sure, which link are we going to provide? readme of github or Terraform resource documentation?

Reply:
readme of github, the one that contains example

Reply:
Done: 
```hcl
Plan: 0 to add, 1 to change, 0 to destroy.
╷
│ Warning: Argument is deprecated
│ 
│   with castai_autoscaler.castai_autoscaler_policies,
│   on castai.tf line 23, in resource "castai_autoscaler" "castai_autoscaler_policies":
│   23:   autoscaler_policies_json = "{}"
│ 
│ use autoscaler_policy_overrides instead. See README for examples: https://github.com/castai/terraform-provider-castai?tab=readme-ov-file#migrating-from-6xx-to-7xx
│ 
│ (and one more similar warning elsewhere)
```

Reply:
nice